### PR TITLE
Add two new step definitions

### PR DESCRIPTION
**Pull request by broeken**
One to check if the response is HAL/JSON, the other to check if the response includes a piece of JSON.

### CODE REVIEW

**Comment by blendle-hubbit** [comment]
Heya!

This PR was just mentioned in the Trello card "[Duplicate subscriptions](https://trello.com/c/oh1kF8cB)" on the [Team Buttons](https://trello.com/b/p6vfusdg) board.

Might be relevant to you, or not...

*Happy Hackin'!*

**Comment by koenbollen** [comment]
:+1:

**Comment by robinvw1** [comment]
:+1:

File: lib/cucumber/blendle/steps/rest_steps.rb [hunk 1]
```diff
@@ -133,3 +123,25 @@
     assert last_response.headers[header], "missing header: #{header}"
   end
 end
+
+# * Then the response at "_embedded/b:subscription-product" should include:
```
Comment (JeanMertz) [inline lib/cucumber/blendle/steps/rest_steps.rb:127]: @broeken just a little nit: this is a public repository, so it makes sense to use generic examples, whereas this is a Blendle-specific example.

**Comment by JeanMertz** [comment]
This PR broke the tests, see: https://app.wercker.com/Blendle/blendle-cucumber-steps/runs/build/57a1f408c4199b0100dde466?step=57a1f40fdb4eec0001fad504

(note: test-reporting wasn't enabled for PRs, it is now, so in new PRs the tests _should_ be reported back to the PR properly)

**Pull request merged**